cmd/init/config.go

```diff
@@ -0,0 +1,63 @@
+package init
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/manifoldco/promptui"
+
+	initConfig "github.com/flyteorg/flytectl/cmd/config/subcommand/init"
+
+	cmdCore "github.com/flyteorg/flytectl/cmd/core"
+	cmdUtil "github.com/flyteorg/flytectl/pkg/commandutils"
+	"github.com/flyteorg/flytectl/pkg/util"
+)
+
+var prompt = promptui.Select{
+	Label: "Select Storage Provider",
+	Items: []string{"S3", "GCS"},
+}
+
+func configInitFunc(ctx context.Context, args []string, cmdCtx cmdCore.CommandContext) error {
+	return initFlytectlConfig(os.Stdin)
+}
+
+func initFlytectlConfig(reader io.Reader) error {
+
+	templateValues := util.ConfigTemplateValuesSpec{
+		Host:     "dns:///localhost:30081",
+		Insecure: true,
+		Template: util.GetSandboxTemplate(),
+	}
+
+	if len(initConfig.DefaultConfig.Host) > 0 {
+		templateValues.Host = fmt.Sprintf("dns:///%v", initConfig.DefaultConfig.Host)
+		templateValues.Insecure = true
```

Comment:
We shouldn't assume Insecure all the time... have you tested this with `demo.nuclyde.io` ? you will probably need to add `--insecure` support as well...